app/server/sql/migrations/2025_10_19_alter_users_columns.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE users ALTER COLUMN first_name DROP NOT NULL;
+ALTER TABLE users ALTER COLUMN last_name DROP NOT NULL;
+ALTER TABLE users ALTER COLUMN email SET NOT NULL;
+ALTER TABLE users ALTER COLUMN phone DROP NOT NULL;
```

Comment:
This can (and should) be combined into one single query to handle it in a transaction and avoid breakage


```suggestion
ALTER TABLE users ALTER COLUMN first_name DROP NOT NULL,
  ALTER COLUMN last_name DROP NOT NULL,
  ALTER COLUMN email SET NOT NULL,
  ALTER COLUMN phone DROP NOT NULL;
```